### DISCUSSION TITLE
Brython on Excel

### DISCUSSION THREAD

**Discussion by maajdl** [brython-dev/brython#2157] [Ideas]
Hello
Excel allows Javascript as a scripting language.
Especially useful for Excel online.
Would it be possible to make Python scripting in Excel available by using Brython ?
Curious about this.
Thanks

**Comment by denis-migdal** [comment]
Hi,

It seems that Excel will support Python:
https://techcommunity.microsoft.com/t5/excel-blog/announcing-python-in-excel-combining-the-power-of-python-and-the/ba-p/3893439

Cordially
